reconcile/utils/helm.py

```diff
@@ -47,3 +52,12 @@ def template(values: Mapping[str, Any]) -> Mapping[str, Any]:
         raise HelmTemplateError(msg)
 
     return yaml.safe_load(result.stdout.decode())
+
+
+def template_from_repo(
+    url: str, path: str, name: str, values: Mapping[str, Any]
+) -> Mapping[str, Any]:
+    with tempfile.TemporaryDirectory() as wd:
+        git.clone(url, wd)
+        print(wd)
```

Comment:
debug print can be removed?

Reply:
https://github.com/app-sre/qontract-reconcile/pull/4472/commits/a667bb7c28da6ea41529a96d4325c5d252262b97